lib/susedistribution.pm

```diff
@@ -232,10 +232,7 @@ sub ensure_installed {
     testapi::assert_script_run('systemctl is-active -q packagekit || (systemctl unmask -q packagekit ; systemctl start -q packagekit)');
     type_string "exit\n";
     my $retries = 5;    # arbitrary
```

Comment:
Couldn't we also drop this definition of `$retries`?

Reply:
yes, of course